Tasks/AzureAppServiceManageV0/task.json

```diff
@@ -146,7 +147,7 @@
                 "EditableOptions": "True"
             },
             "required": true,
-            "visibleRule": "Action != Swap Slots && SpecifySlot = true"
+            "visibleRule": "Action = Delete Slot || SpecifySlot = true"
```

Comment:
Since combination of && and || is not supported, we can either go with this or introduce another task input. My suggestion is that we keep it this way, so that it is optimized in yaml flow.